src/features/auth/pages/Login.test.tsx

```diff
@@ -48,15 +53,13 @@ describe('<Login/>', () => {
       </NavigationContainer>
     )
 
-    fireEvent.press(getByText('Connexion'))
+    await act(() => fireEvent.press(getByText('Connexion')))
 
-    await act(async () => {
-      const HomeMessage = await waitFor(() => queryByText('Home message'))
-      expect(HomeMessage).toBeNull()
+    act(async () => {
+      const failureMessage = await waitFor(() =>
+        queryByText('Échec de la connexion au Pass Culture')
```

Comment:
Idem, les `string littéraux` dans les tests ne sont vraiment pas idéaux à mon avis.

Reply:
C'est le concept de react-testing^^

Reply:
Je ne suis pas tout à fait de cet avis

Reply:
La philosophie de react-testing est d'écrire tes tests de la même manière que les utilisateurs utilisent l'app. Hors, les utilisateurs voient les wordings tel quelles, pas des ids.
Si cela peut te rassurer, pour le moment les wordings sont égaux aux ids^^

Reply:
Je suis assez d'accord avec Ben

Reply:
Je suis plutôt convaincu, c'est un peu dommage pour la maintenabilité ! Merci !

Reply:
Je rejoins un peu Gireg sur ce point et je pense que c'est une discussion qu'on devrait avoir ensemble.

utiliser un json pour regrouper au même endroit toutes les clés de traduction pour n'avoir qu'un endroit à modifier au besoin est un confort pour ceux qui auront à maintenir l'application:

`queryByText(traduction.EchecConnexionPassculture)`

Reply:
Mais du coup avec ce fonctionnement, on ne détecte jamais les erreurs de typo ou autre. Il me semblait que les tests ne servent pas à factoriser quoique ce soit mais au contraire être redondant s'il le faut, pour coller un maximum au rendu final.

Reply:
> on ne détecte jamais les erreurs de typo ou autre.

Il y a quand même, entre autre
* Les relectures de PR
* Les reviews PO testing
* Les reviews staging

L'argument n'est pas de factoriser, mais de les rendre plus maintenables. (ie de ne pas avoir à les corriger quand on change des wordings) 
Quand tu testes si une traduction est présente, c'est parce que tu veux t'assurer que la logique a été bonne.
On peut aussi se servir des tests pour nous assurer que le wording est bon (enfin ça ajoute juste une couche légère supplémentaire de vérification, si tu me demandes de coder un test avec une wording littéral, je vais copier coller ce que j'ai mis dans la feature)

Reply:
L'avis du maintainer de notre outils de test sur cela : https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text
Note : cet article review les erreurs que Kent a tendance à voir dans les projets qui utilisent `react-testing-library`.

Reply:
Ok pour la détection de erreurs avec les relectures de PR, en revanche les reviews PO testing et staging signifient que l'erreur a déjà été mergée, est-ce qu'on ne perdra pas plus de temps à créer un ticket jira pour une typo et refaire tout le process de dev associé, plutôt que de corriger un petit test qui casse en local (qui ne prend pas plus d'une minute à mon avis) ?
Mais bon, je comprend le point de tester uniquement la logique, peut-être que je suis trop pessimiste et que les erreurs de wordings n'arriveront jamais :)

Reply:
En soit les deux ont un sens, 
C'est vrai qu'un changement wording peut impliquer des changements côté logique, et donc qu'on veut le savoir absolument. 
Je serais ok pour les 2